test/e2e/utils.go

```diff
@@ -245,6 +247,11 @@ func printTestStackTrace() {
 }
 
 func undeployJaegerInstance(jaeger *v1.Jaeger) bool {
+	if saveLogs {
+		logFileName := strings.ReplaceAll(t.Name(), "/", "-") + ".log"
+		writePodLogToFile(jaeger.Namespace, "app=jaeger", "jaeger", logFileName)
```

Comment:
Could we try `app.kubernetes.io/part-of` which should also have the `jaeger` value, or `app.kubernetes.io/managed-by` value `jaeger-operator`?
Although we generate the `app` label currently, it is not one of the recommended k8s labels, so may be removed in the future.

Reply:
Sure, I can change that.